libpdbg/device.c

```diff
@@ -600,10 +600,11 @@ uint64_t pdbg_target_address(struct pdbg_target *target, uint64_t *out_size)
 
 	u32 na = dt_n_address_cells(target);
 	u32 ns = dt_n_size_cells(target);
-	u32 n;
+#ifndef NDEBUG
+	u32 n = (na + ns) * sizeof(u32);
+#endif
 
 	p = dt_require_property(target, "reg", -1, &len);
-	n = (na + ns) * sizeof(u32);
 	assert(n <= len);
```

Comment:
This check is also required for address translation to work correctly.  Dropping this hunk from patch.